playbooks/files/create_cce_cluster.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+
+import openstack
+from otcextensions import sdk
+
+openstack.enable_logging(True, http_debug=True)
+
+# An 'otc' is a cloud connection with name 'otc' configured in the clouds.yaml
+conn = openstack.connect()
+
+# Register OTC Extensions
+sdk.register_otc_extensions(conn)
+
+cluster = conn.cce.find_cluster(sys.argv[1])
+dict = {
+    'metadata': {
+        'name': sys.argv[1]
+    },
+    'spec': {
+        'type': 'VirtualMachine',
+        'version': 'v1.11.7-r2',
+        'hostNetwork': {
+            'vpc': sys.argv[2],
+            'subnet': sys.argv[3]
+        },
+        'flavor': 'cce.s1.small',
+        'containerNetwork': {
+            'mode': 'overlay_l2',
+            'cidr': '172.16.0.0/16'
+        }
+    }
+}
+
+if (cluster is None):
+  cluster = conn.cce.create_cluster(**dict)
+  #time.sleep(720)
+  conn.cce.wait_for_cluster(cluster, status='Available', failures=None,
```

Comment:
there is now `conn.cce.wait_for_job(cluster.job.id)` in otcextensions. Let's switch to it

Reply:
does not work because cluster.job.id does not exist from response.